app/assets/stylesheets/licence-finder.scss

```diff
@@ -2,8 +2,14 @@
 @import "_typography";
 @import "_css3";
 
+@import "govuk_publishing_components/all_components";
+
 /* see multi-step.css.erb in the static repo for most of these styles */
 
+#content {
+  margin-bottom: $gutter;
```

Comment:
I assume this is to add some space between the page content and the feedback component? We may need to revisit this as I think some top margin should just be added to the component itself.

Reply:
Yup. Without it there's 0 spacing between the elements (in other apps there's at least some).